spyder/app/mainwindow.py

```diff
@@ -905,6 +906,17 @@ def create_edit_action(text, tr_text, icon):
             self.findinfiles = FindInFiles(self)
             self.findinfiles.register_plugin()
 
+        # Load other plugins (former external plugins)
+        other_plugins = ['breakpoints', 'profiler', 'pylint']
+        for plugin_name in other_plugins:
+            if CONF.get(plugin_name, 'enable'):
+                module = importlib.import_module(
+                        'spyder.plugins.{}'.format(plugin_name))
+                plugin = module.PLUGIN_CLASS(self)
+                if plugin.check_compatibility()[0]:
+                    self.thirdparty_plugins.append(plugin)
+                    plugin.register_plugin()

```

Comment:
This shouldn't be necessary now.

Reply:
I think this is necessary, It I remove it I should move the logic of register_plugin to the mainwindow (and personally I don't like that approach)